handlers/handlers.go

```diff
@@ -121,3 +221,79 @@ func getRequestBody(request *http.Request) []byte {
 
 	return body
 }
+
+
+// Token Util
+const oneWeekInMinutes = 60 * 24 * 7
+const cerealNotesCookieName = "CerealNotesToken"
+
+func parseTokenFromString(tokenString string) (*jwt.Token, error) {
+	// Parse the token
+	token, err := jwt.ParseWithClaims(
+		strings.TrimSpace(tokenString),
+		&CerealNotesClaims{},
+		func(token *jwt.Token) (interface{}, error) {
```

Comment:
golang doesn't through if you have unused function parameters. I think its so that you don't have weird situation when you implement interfaces and don't use all the parameters.